yarn.lock

```diff
@@ -1328,20 +1328,23 @@
     "@types/istanbul-reports" "^1.1.1"
     "@types/yargs" "^12.0.9"
 
-"@kiwicom/account-native@^0.0.1":
-  version "0.0.1"
-  resolved "https://registry.yarnpkg.com/@kiwicom/account-native/-/account-native-0.0.1.tgz#f9d9a0504e4d4373cf1cdf5b402b5e1ab375ebaa"
-  integrity sha512-Ggu+1g1tS8hLOloUD+Bt9SdrpiybWmDUJ+k7mWvlb1emfEcRmob3+ANbSA/uXRiHdx204OcSmtEEzZXWX+QQaA==
+"@kiwicom/account-native@^0.0.4":
+  version "0.0.4"
+  resolved "https://registry.yarnpkg.com/@kiwicom/account-native/-/account-native-0.0.4.tgz#300411e7d29ebb85e20bf8dbffbf9581df788f9b"
+  integrity sha512-N03UKZR6+CLQTZX9s8/JE3Six1vtxqR07TtUO4wSRxrp/Ehq84sZ3te4FF2bEdglpJnmCXfgX+jhFZXy21uNoA==
   dependencies:
     "@kiwicom/graphql-global-id" "^0.10.0"
     "@kiwicom/graphql-utils" "^0.8.0"
+    "@kiwicom/js" "^0.13.0"
+    "@kiwicom/orbit-components" "^0.53.0"
     "@kiwicom/orbit-design-tokens" "^0.6.5"
     "@kiwicom/relay" "^3.2.0"
     buffer "^5.2.1"
     dataloader "^1.4.0"
     date-fns "^1.30.1"
     graphql "^14.4.2"
     graphql-iso-date "^3.6.1"
+    npm-registry-client "^8.6.0"
```

Comment:
Not related completely to this PR, but why is `npm-regristry-client` a dependency of this package?

Reply:
I made a note of it, and I will improve it in the next release of account-native package.